src/email/send.service.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('Email send service', function() {
+    var EmailSendService;
+
+    beforeEach(module('motech-email'));
+    beforeEach(inject(function( _EmailSendService_) {
+        EmailSendService = _EmailSendService_;
+    }));
+
+    it('should execute sendEmail with good parameters, two times', function() {
+        var meesage= { name : "Damian" };
```

Comment:
`message` and add space before equals sign and change "Damian" to an English name like John etc.